.github/workflows/release-series-images.yml

```diff
@@ -95,33 +87,22 @@ jobs:
         # which is required for build-push-action
         IFS=,
         echo "::set-output name=taglist::${tag_list[*]}"
-
+    
     - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v1
-
-    - name: Log in to Docker Hub
-      uses: docker/login-action@v1
-      if: github.event_name != 'pull_request' && startsWith(github.repository, 'opensciencegrid/')
       with:
-        username: ${{ secrets.DOCKER_USERNAME }}
-        password: ${{ secrets.DOCKER_PASSWORD }}
-
-    - name: Log in to OSG Harbor
-      uses: docker/login-action@v1
-      if: github.event_name != 'pull_request' && startsWith(github.repository, 'opensciencegrid/')
-      with:
-        registry: hub.opensciencegrid.org
-        username: ${{ secrets.OSG_HARBOR_ROBOT_USER }}
-        password: ${{ secrets.OSG_HARBOR_ROBOT_PASSWORD }}
-
-    - name: Build and push Docker image
+        driver: docker  # If not set to docker driver, it will default to docker-container
+                        # when using load for the build-push-action.
+    
+    - name: Build and push hosted-ce and osg-ce-condor images
       uses: docker/build-push-action@v2.2.2
       with:
+        load: true
         push: ${{ github.event_name != 'pull_request' && startsWith(github.repository, 'opensciencegrid/') }}
-        context: .
+        context: ${{matrix.dockercontext}}
+        file: ${{matrix.dockerfile}}
```

Comment:
This works but it's a little funny. The `image`, `dockerfile`, and `dockercontext` are all basically the same value with just a little extra text around the latter two. If you find yourself repeating/copy pasting stuff like this, it's an indication that it can be [refactored](https://en.wikipedia.org/wiki/Code_refactoring#Motivation) to be simplier!